src/popup/popup.css

```diff
@@ -20,15 +20,17 @@ button {
   font-size: 14px;
   color: #000000;
   background-color: #ffffff;
-  border: 1px solid #000000;
+  /* border: 1px solid #000000; */
```

Comment:
Nit, in general it is better to do not comment out the parts that you want to replace (1), just remove them, the diff will already tell us what was the patch replaced.

----

(1): in this case it is easy to spot them, but in a bigger diff some of the commented parts may go unnoticed and may land on master by mistake